src/EA.Weee.Web/Views/Shared/Producer/ViewOrganisation/ContactDetails.cshtml

```diff
@@ -80,6 +80,22 @@
                 </tbody>
             </table>
         </div>
+        @if (Model.IsInternalAdmin)
+        {
+            <div>
```

Comment:
@will-hope98  not sure how easy this is. But is it possible to put this into the _OrganisationDetailsTabs.cshtml to save having it in each view.

Reply:
@JohnPadgettCivica Would have to create a new partial I think. if you prefer to do it that way

Reply:
@will-hope98 OK as this is at the bottom of the page. Stick it in a partial then as that seems to be best way of re-using